src/components/Heading/Heading.tsx

```diff
@@ -1,18 +1,17 @@
 import { camelCase } from "lodash-es";
```

Comment:
since `Title` is the modern replacement, we should at some point deprecate this component by adding a deprecation log, documentation banner, and codemod.

let's create a dedicated task for that

Reply:
Adding to backlog